modules/electrophysiology_browser/test/TestPlan.md

```diff
@@ -0,0 +1,23 @@
+## Electrophysiology Browser test plan
+	
+### A. Electrophysiology Browser front page
+1. User can load Electrophysiology Browser module front page IFF has either permission:
```

Comment:
```suggestion
1. User can load Electrophysiology Browser module front page if and only if user has either permission:
```

Reply:
I just realized today what IFF meant in a review comment from Dave so it might be best to use long way of saying it.

Reply:
Huh? I didn't make any review comments today explaining the word..

Reply:
LOL, I thought I saw your name but maybe not. (I saw that in another PR, not this one, or maybe I just dreamt it and had a sudden revelation of what IFF meant :P)